### PR TITLE
Feature/liken

### PR DESCRIPTION
**Pull request by SimonPlanje**
Styling is not perfect yet, and we have to discuss the megalike button.

### CODE REVIEW

File: views/css/style.css [hunk 1]
```diff
@@ -240,7 +240,7 @@ Like form
   margin: 2em;
 }
 
-#likeForm input {
+/* #likeForm input {
```
Comment (Vuurvos1) [inline views/css/style.css:243]: Maybe just delete this commented code

File: views/components/likeForm.ejs [hunk 2]
```diff
@@ -1,25 +1,45 @@
...
+      <h2 class="age"><%= data.age %></h2>
+      <p>
+        Hallo, dit is de beschriving van Kees
+      </p>
+    </div>
   </div>
+
+  <form class="addData" action="/like" method="post">
+    <input type="text" class="userId" name="id" value="<%= data._id %>" />
+    <input name="review" value="like" id="like" type="submit" />
```
Comment (Vuurvos1) [inline views/components/likeForm.ejs:18]: Move type to the start for better readability

File: views/components/likeForm.ejs [hunk 3]
```diff
@@ -1,25 +1,45 @@
...
+    <label for="like" class="like"></label>
+  </form>
+
+  <form class="addData" action="/like" method="post">
+    <input type="text" class="userId" name="id" value="<%= data._id %>" />
+    <input name="review" value="megalike" id="megalike" type="submit" />
```
Comment (Vuurvos1) [inline views/components/likeForm.ejs:24]: Also here

File: views/components/likeForm.ejs [hunk 4]
```diff
@@ -1,25 +1,45 @@
...
+    <label for="megalike" class="megalike"></label>
+  </form>
+
+  <form class="addData" action="/like" method="post">
+    <input type="text" class="userId" name="id" value="<%= data._id %>" />
+    <input name="review" value="dislike" id="dislike" type="submit" />
```
Comment (Vuurvos1) [inline views/components/likeForm.ejs:30]: and here D:

File: routes/likepage.js [hunk 5]
```diff
@@ -1,8 +1,76 @@
...
+    res.render('./liken.ejs', { data: users[index.number] });
+  }
+});
+
+//***************************************//
+//******* WHEN FORM GETS POSTED *********//
+//***************************************//
+
+router.post('/like', (req, res) => {
+  if (req.body.review === 'like') {
```
Comment (Vuurvos1) [inline routes/likepage.js:32]: This part looks quite long and inefficient, might be wrong about that though. If you think this is the right solution then this is fine

Comment (Vuurvos1) [reply routes/likepage.js:32]: Also this branch will have merge conflicts so make sure to merge the master into this one!

**Comment by SimonPlanje** [comment]
I checked all the reviews, I think i it okay to merge now :)

**Pull request merged**